.rubocop-rspec.yml

```diff
@@ -12,7 +12,7 @@ RSpec/NestedGroups:
 RSpec/ExampleLength:
   Exclude:
     - 'spec/features/**/*'
-  Max: 7
+  Max: 20
```

Comment:
From my understanding, this would not do what you described. 
Feature specs are already excluded from this cop (things in `spec/features`) 
This would allow all examples to be 20 lines long, also unit tests, since the `Max` configures the entire cop. 

I think we could just add more Exclude paths if we want to care for other kinds of `feature` specs (like `spec/integration/**/*`)

Reply:
True 😂 

Let's leave this for now.